src/urllib3/util/ssl_.py

```diff
@@ -278,7 +280,7 @@ def create_urllib3_context(
         context.check_hostname = False
         context.verify_mode = cert_reqs
 
-    if hasattr(context, "hostname_checks_common_name"):
+    if HAS_NEVER_CHECK_COMMON_NAME:
         context.hostname_checks_common_name = False
```

Comment:
Should we also defensively swallow `AttributeError` on this line?

Reply:
I'm reluctant to do this: this behavior is clearly documented in Python 3.7+ and PyPy and CPython share the same simple implementation.

This also won't be just False in 2.0 now that we know exactly what CPython and OpenSSL versions work correctly.